### ISSUE TITLE
Escaped strings make `--autogenerate` always autogenerate

### ISSUE THREAD

**Issue by DanCardin** [olirice/alembic_utils#58]
I produced a view a little something like this:

```python
query = "SELECT concat('https://something/', column) FROM table"
some_view = PGView(schema="public", signature="example", definition=query)
```

Which generated a revision something like so
```python
    public_something = PGView(
        schema="public",
        signature="something",
        definition="SELECT concat('https\\://something/', column) FROM table",
    )
    op.replace_entity(public_something)
```

Notably the `\\:` section, which seems like an erroneous double escape or something.

If I run this upgrade, and re `--autogenerate`, it generates a new revision with `\\\\:`.

If you remove the `\\` generated in the first place, all seems well.

**Comment by olirice** [comment]
Thank you for reporting, I was able to reproduce the error for your example

**Comment by olirice** [comment]
a fix is out in `alembic_utils==0.7.2`
please let me know if you have any trouble with it

**Comment by DanCardin** [comment]
Works for me, thanks!
